### PR TITLE
Merge next on release

### PR DESCRIPTION
**Pull request by spolti**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: businesscentral-monitoring/branch-overrides.yaml [hunk 1]
```diff
@@ -14,20 +14,20 @@ modules:
...
-        ref: EAP_7410_CR2
+        ref: EAP_7412_CR3
     - name: wildfly-cekit-modules
       git:
         url: https://github.com/wildfly/wildfly-cekit-modules.git
         ref: 0.18.11
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: 7.13.x
```
Comment (rgdoliveira) [inline businesscentral-monitoring/branch-overrides.yaml:29]: shouldn't this keep as 7.13.x as we are moving to 7.13.x branch?

**Review by desmax74 — APPROVED** [review]

File: businesscentral-monitoring/branch-overrides.yaml (hunk 1 above)
Comment (spolti) [reply businesscentral-monitoring/branch-overrides.yaml]: correct, updated.
thanks.

File: quickstarts/router-ext/README.md [hunk 2]
```diff
@@ -36,11 +36,15 @@ $ cd /tmp/smartrouter
 To extract the `openshift-launch.sh` file from the official Smart Router image, enter the following command:
 
 ```bash
-$ docker run --rm registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.13.3 \
+$ docker run --rm registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.13.4 \
     cat /opt/rhpam-smartrouter/openshift-launch.sh > openshift-launch.sh
 ```
 
+<<<<<<< HEAD
```
Comment (rgdoliveira) [inline quickstarts/router-ext/README.md:43]: It seems there is a conflict here

File: quickstarts/hello-rules-multi-module/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ## Red Hat Process Automation Manager KIE Server decisions Quickstart - Multi module
 
-This quickstart is intended to be used with the [RHPAM Kie Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.3-1.GA/kieserver) image.
+This quickstart is intended to be used with the [RHPAM Kie Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.3.GA/kieserver) image.
```
Comment (desmax74) [inline quickstarts/hello-rules-multi-module/README.md:3]: 7.13.4 ?

File: quickstarts/hello-rules-multi-module/README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
 ## How to use it?
 
-To deploy the Hello Rules demo you can use the [rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.13.3-1.GA/templates/decision/rhdm713-prod-immutable-kieserver.yaml) application template.
+To deploy the Hello Rules demo you can use the [rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.13.3.GA/templates/decision/rhdm713-prod-immutable-kieserver.yaml) application template.
```
Comment (desmax74) [inline quickstarts/hello-rules-multi-module/README.md:7]: 7.13.4 ?

File: quickstarts/hello-rules/README.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 ## Red Hat Process Automation Manager KIE Server decisions Quickstart
 
-This quickstart is intended to be used with the [RHPAM KIE Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.3-1.GA/kieserver) image.
+This quickstart is intended to be used with the [RHPAM KIE Server](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.3.GA/kieserver) image.
```
Comment (desmax74) [inline quickstarts/hello-rules/README.md:3]: 7.13.4 ?

File: quickstarts/hello-rules/README.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 
 ## How to use it?
 
 The template below will be used for this quickstart:
 
-[rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.13.3-1.GA/templates/decision/rhdm713-prod-immutable-kieserver.yaml) application template.
+[rhdm713-prod-immutable-kieserver](https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.13.3.GA/templates/decision/rhdm713-prod-immutable-kieserver.yaml) application template.
```
Comment (desmax74) [inline quickstarts/hello-rules/README.md:9]: 7.13.4 ?

**Comment by spolti** [comment]
Thanks @desmax74, fixed the old versions.

**Review by rgdoliveira — APPROVED** [review]

**Pull request merged**